app/src/main/java/cm/aptoide/pt/presenter/MainView.java

```diff
@@ -27,4 +28,10 @@
   void showUpdatesNumber(Integer updates);
 
   void hideUpdatesBadge();
+
+  Observable<PermissionService> autoUpdateDialogCreated();
+
+  void requestAutoUpdate();
+
+  void handleAutoUpdateResult(boolean installFailed);
```

Comment:
This method is strange, I think this logic of what to show should be in the presenter, view would only know how to show.